src/node/markdownToVue.ts

```diff
@@ -189,13 +189,14 @@ export async function createMarkdownToVueRenderFn(
           return ignore.test(url)
         }
         if (typeof ignore === 'function') {
-          return ignore(url)
+          return ignore(url, fileOrig)
         }
         return false
       })
     }
 
-    if (links) {
+    const { ignoreDeadLinks = true } = siteConfig
+    if (links && ignoreDeadLinks !== true) {
```

Comment:
why is this added?
also destructuring here is bit confusing since ignoreDeadLinks defaults to false.